.github/workflows/ci.yml

```diff
@@ -26,13 +26,13 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
-        pip install -r dev_requirements.txt
+        pip install pipenv
+        pipenv install --dev
     - name: Lint with Pylint
       run: |
-        pylint msgraphcore --disable=W --rcfile=.pylintrc
+        pipenv run pylint msgraphcore --disable=W --rcfile=.pylintrc
     - name: Test with pytest
       run: |
-        coverage run --omit 'venv/*' -m unittest discover tests/unit
-        coverage run --omit 'venv/*' -m unittest discover tests/integration
-        coverage html
+        pipenv run coverage run --omit 'venv/*' -m unittest discover tests/unit
```

Comment:
This shouldn't be necessary, we don't a venv folder.

Reply:
Right.